guides/micronaut-jms-aws-sqs/groovy/src/test/groovy/example/micronaut/MicronautguideTest.groovy

```diff
@@ -0,0 +1,46 @@
+package example.micronaut
+
+import io.micronaut.core.annotation.NonNull;
+import io.micronaut.test.extensions.junit5.annotation.MicronautTest;
+import io.micronaut.test.support.TestPropertyProvider;
+import jakarta.inject.Inject;
+import org.junit.jupiter.api.AfterAll;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.TestInstance;
+import org.testcontainers.junit.jupiter.Testcontainers;
+
+@Testcontainers(disabledWithoutDocker = true)
+@TestInstance(TestInstance.Lifecycle.PER_CLASS)
+@MicronautTest
+class MicronautguideTest implements TestPropertyProvider {
+
+    @Inject
+    DemoController demoController
+
+    @Inject
+    DemoConsumer demoConsumer
+
+    @Test
+    void testItWorks() {
+        int messageCount = demoConsumer.getMessageCount()
+        assert messageCount == 0
+        demoController.publishDemoMessages()
```

Comment:
I would prefer if we injected a HTTP Client and invoke the controller from the outside. 

Replace 

```
@Inject
DemoController demoController
```

with 

```
@Inject
@Client("/")
HttpClient httpClient
````

replace: 

`demoController.publishDemoMessages()`

with: 

`httpClient.toBlocking().exchange(HttpRequest.POST("/demo", Collections.emptyMap())`

Reply:
Will do it.